alchemy-web/docs/getting-started.md

```diff
@@ -1,40 +1,100 @@
 ---
 title: Getting Started with Alchemy
-description: Quick start guide to using Alchemy, the TypeScript-native Infrastructure-as-Code library. Learn how to install, create resources, and manage their lifecycle.
+description: Quick start guide to using Alchemy, the TypeScript-native Infrastructure-as-Code library. Deploy your first Cloudflare Worker with type-safe infrastructure code.
 ---
 
 # Getting Started with Alchemy
 
+This guide walks you through deploying your first Cloudflare Worker using Alchemy, demonstrating Infrastructure-as-Code principles with a real-world, practical example.
+
 > [!TIP]
-> Read [What is Alchemy](./what-is-alchemy.md) to get an overview of Alchemy and how it's different than tradtional IaC
+> Read [What is Alchemy](./what-is-alchemy.md) to get an overview of Alchemy and how it's different than traditional IaC
+
+## Prerequisites
+
+You'll need:
+- [Node.js](https://nodejs.org/) or [Bun](https://bun.sh/)
```

Comment:
This should be driven off a code group that is either bun, node or deno rather than saying "or"